jobs/build.Jenkinsfile

```diff
@@ -477,6 +477,27 @@ lock(resource: "build-${params.STREAM}") {
                     """)
                 }
             }
+            if (!is_mechanical) {
+                stage("Push KubeVirt image") {
+                    withCredentials([file(credentialsId: 'oscontainer-secret', variable: 'OSCONTAINER_SECRET')]) {
+                            // Push the KubeVirt containerdisk to quay.io/coreos-assembler/fedora-coreos.
+                            // The image gets a moving tag which correlates to the stream name to represent the latest state.
+                            // Users should not reference these tags directly for production workloads. They should either clone
+                            // the image to have a snapshot from the current stream state, or reference it via digest to get
+                            // reproducible results.
```

Comment:
This is contrary to Fedora CoreOS policy.  We strongly discourage running any release other than the latest releases of the production streams (`stable`/`testing`/`next`).  If users want to do otherwise, we don't use technical means to stop them, but we don't help them either.  Let's drop this language, and have the docs just point to the stream tags.

Reply:
Removed and updated the org to `coreos`.